### ISSUE TITLE
Accessibility Concerns with Sign-Up Form Visibility and Keyboard Focus

### ISSUE THREAD

**Issue by ChalkandFeather** [hjtrhodes/bald-bible#44]
**Issue:**
The sign-up button in the navbar successfully navigates users to the sign-up page. However, a design feature involving an animation on the sign-up page causes the sign-up form to be hidden initially. The form remains concealed until in a hover state, creating an accessibility concern for users who rely on keyboard navigation.

**Steps to Reproduce:**
Navigate to the sign-up page using the sign-up button in the navbar.
Observe the sign-up form visibility, especially without hover interaction.
Attempt to interact with the sign-up form using keyboard navigation.

**Expected Behaviour:**
The sign-up form should be immediately visible upon entering the sign-up page, ensuring that users, including those who navigate via keyboard, can easily access and complete the form.

**Actual Behavior:**
The sign-up form is initially hidden, and its visibility is dependent on hover interactions. This poses challenges for keyboard users as they are unable to access the form without visual cues.

Priority: P3 (significant)
Severity (1 to 5) 3 (medium)

**Additional Information:**
This issue has been observed consistently on [browser name and version].
Confirm whether the issue persists across different browsers.
Check if the issue is present in both mobile and desktop views.

**Recommendation**
Ensure that the sign-up form is initially visible without relying on hover interactions. 
Avoid using design elements that hide or reveal essential interactive elements solely based on hover states. This approach can be problematic for users with different interaction methods, including keyboard navigation.
Use ARIA (Accessible Rich Internet Applications) attributes to enhance the accessibility of dynamic content. For instance, consider using aria-hidden appropriately or modifying ARIA attributes dynamically to reflect the current state of the form.

**Impact:**
Hinders accessibility for keyboard users.
May lead to difficulties in signing up for users relying on assistive technologies.

**Comment by ChalkandFeather** [comment]
Focus on Navbar 
<img width="1280" alt="Screenshot 2024-01-10 at 21 34 49" src="https://github.com/hjtrhodes/bald-bible/assets/114813829/2c116b03-103f-4207-a47b-8cdf462c3bba">

Hover on body triggers animation to reveal form
<img width="1280" alt="Screenshot 2024-01-10 at 21 35 00" src="https://github.com/hjtrhodes/bald-bible/assets/114813829/bc41dc81-d75c-46f6-8fc3-3e18adbf7112">
